extensions/amp-dailymotion/0.1/amp-dailymotion.js

```diff
@@ -264,11 +274,17 @@ class AmpDailymotion extends AMP.BaseElement {
       'info',
     ];
 
-    settings.forEach(setting => {
-      this.addQueryParam_(setting, query);
+    let explicitParams = dict();
+
+    explicitParamsAttributes.forEach(explicitParam => {
+      this.addDictParam_(explicitParam, explicitParams);
```

Comment:
no need for `addDicParam_` we can do:
1- add `data-` to all the `explicitParamsAttributes` values
2- import `addParamToUrl`
3- inline the whole thing:
```
explicitParamsAttributes.forEach(explicitParam => {
   const val = this.element.getAttribute(explicitParam);
   if (val) {
      addParamToUrl(iframeSrc, explicitParam, val);
   }
});
```